### PR TITLE
Update after review

### PR DESCRIPTION
**Pull request by Matteo04052017**
Added page "10 things you should know about CORBA"
Review of page HDB++ for beginner for english and correction for drawing.

### CODE REVIEW

File: source/overview/corba.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+10 things you should know about CORBA
+=====================================
+
+1.  ***You don’t need to know CORBA to work with TANGO***
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest
+        **B**\ roker **A**\ rchitecture and it is a standard defined by
+        the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object
+        Management Group (OMG)
```
Comment (piogor) [inline source/overview/corba.rst]: Hi @Matteo04052017, please make points indentation correct, like below:
```
2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest
    **B**\ roker **A**\ rchitecture and it is a standard defined by
    the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object
    Management Group (OMG)
``` 
(Subsequent lines of the same point should be spaced with such number of space so it starts below latter of the first line)

File: source/overview/corba.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+10 things you should know about CORBA
+=====================================
+
+1.  ***You don’t need to know CORBA to work with TANGO***
```
Comment (piogor) [inline source/overview/corba.rst]: Please remove additional asterixes (two on both sides of text will do the work).

Comment (piogor) [inline source/overview/corba.rst]: Of course, I may do the changes by myself if you wish so :)

Comment (Matteo04052017) [inline source/overview/corba.rst]: No no Piotr, don't worry! I'll do it! It is the best to learn how to do it... ;)

File: source/overview/corba.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  You don’t need to know CORBA to work with TANGO
```
Comment (bourtemb) [inline source/overview/corba.rst]: I think it was nice to have this sentence in bold

Comment (Matteo04052017) [inline source/overview/corba.rst]: agree

File: source/overview/corba.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object Management Group (OMG)
```
Comment (piogor) [inline source/overview/corba.rst]: This is how to create a link with text other than URL:
```
`Object Management Group (OMG) <https://en.wikipedia.org/wiki/Object_Management_Group>`_
```
@Matteo04052017, please correct that as well.

File: source/overview/corba.rst [hunk 5]
```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the `Object Management Group (OMG) <www.omg.org>`_
```
Comment (piogor) [inline source/overview/corba.rst]: @Matteo04052017 , To make an external link works you should always add http:// like:
`Object Management Group (OMG) <http://www.omg.org>`_

Comment (piogor) [inline source/overview/corba.rst]: You may always try to build documentation locally before commit: `sphinx-build source/ build`, then open `build/index.html` with your browser and navigate to your document  ;-)

Comment (Matteo04052017) [inline source/overview/corba.rst]: hehe I tried to compile but I was unsuccessful... this was the error:

# Sphinx version: 1.6.2
# Python version: 2.7.11 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\sphinx\cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "c:\python27\lib\site-packages\sphinx\application.py", line 168, in __init__
    confoverrides or {}, self.tags)
  File "c:\python27\lib\site-packages\sphinx\config.py", line 150, in __init__
    execfile_(filename, config)
  File "c:\python27\lib\site-packages\sphinx\util\pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "c:\python27\lib\site-packages\six.py", line 699, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "conf.py", line 18, in <module>
ImportError: No module named IPython.sphinxext

Of course the problem is that I didn't spend enough time on it.. Sorry about that.

**Comment by piogor** [comment]
Hi Matteo, I suppose you need to install IPython, command: pip install ipython

 

Br.

Piotr

**Comment by piogor** [comment]
@Matteo04052017, I am going to delete the branch.

**Pull request merged**